src/ImageRunModal.jsx

```diff
@@ -861,7 +861,7 @@ export class ImageRunModal extends React.Component {
                                   <Popover aria-label={_("CPU Shares help")}
                                       enableFlip
                                       bodyContent={_("CPU shares determine the priority of running containers. Default priority is 1024. A higher number prioritizes this container. A lower number decreases priority.")}>
-                                      <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                      <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-125816-eb1c0f76-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)